Java/AutoML/automl-component/src/test/java/com/automl/component/dao/DatasetDAOTest.java

```diff
@@ -0,0 +1,63 @@
+package com.automl.component.dao;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import com.automl.component.model.Dataset;
+
+import io.github.qingguox.json.JacksonUtils;
+
+/**
+ * @author wangqingwei
+ * Created on 2022-08-10
+ */
+@SpringBootApplication(scanBasePackages = "com.automl")
+public class DatasetDAOTest implements CommandLineRunner {
+
+    private static final Logger logger = LoggerFactory.getLogger(DatasetDAOTest.class);
+
+    @Autowired
+    private DatasetDAO datasetDAO;
+
+
+    @Override
+    public void run(String... args) throws Exception {
+//        insertOneCase();
+        getAll();
+    }
+
+    private void insertOneCase() {
```

Comment:
unused function insertOneCase

Reply:
<img width="812" alt="image" src="https://user-images.githubusercontent.com/48853704/185661620-7c59c195-7347-4657-be3b-cc16aeb6f923.png">
see this there,  we can flexible use of method testing. this test is a web application.

Reply:
However, we'll investigate the use of jdbcTest and Configuration later

Reply:
> see this there, we can flexible use of method testing. this test is a web application.
Got it!